renovate.json5

```
@@ -4,5 +4,12 @@
     "github>giantswarm/renovate-presets:default.json5",
     // Go specific config - https://github.com/giantswarm/renovate-presets/blob/main/lang-go.json5
     "github>giantswarm/renovate-presets:lang-go.json5",
+    // Pin to the SHA of a GitHub action.
+    "helpers:pinGitHubActionDigests"
   ],
+  "github-actions": {
+   "fileMatch": [
+     ".*.yaml.template$",
+   ],
+  },
```

Just a little formatting
```suggestion
  "github-actions": {
    "fileMatch": [
      ".*.yaml.template$",
    ],
  },
```